metashare/api/models.py

```diff
@@ -131,6 +132,8 @@ def instance_url(self):
 
     @property
     def sf_username(self):
+        if settings.DEVHUB_USERNAME and self.use_global_devhub:
+            return settings.DEVHUB_USERNAME
```

Comment:
DEVHUB_USERNAME should be the fallback if a dev hub username isn't found from one of the user-specific sources.